ws-server/src/server.rs

```diff
@@ -240,11 +240,13 @@ async fn background_task(
 	conn_id: ConnectionId,
 	methods: Methods,
 	resources: Resources,
-	_max_request_body_size: u32,
+	max_request_body_size: u32,
 	stop_server: StopMonitor,
 ) -> Result<(), Error> {
 	// And we can finally transition to a websocket background_task.
-	let (mut sender, mut receiver) = server.into_builder().finish();
+	let mut builder = server.into_builder();
+	builder.set_max_message_size(max_request_body_size as usize);
```

Comment:
This is for both incoming and outgoing traffic? The soketto docs are not clear on that.